source/fundamentals/bson.txt

```diff
@@ -0,0 +1,301 @@
+=================
+Working with BSON
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about how the Go Driver handles conversions
+between BSON and Go objects. The process of converting a Go object to
+BSON is called **marshalling**, while the reverse process is called **unmarshalling**. 
```

Comment:
nit: The conventional spelling in Go docs always uses one "L" in these words. Consider changing the spelling to always use one "L".

E.g.
* marshaling
* marshaler
* unmarshaling
* unmarshaler

Reply:
Hmm the API Documentation seems to use two L's: https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.7.0/bson#hdr-Marshalling_and_Unmarshalling